upgrade/1.4.4/CSM-Only.md

```diff
@@ -0,0 +1,136 @@
+# CSM Only Upgrade
+
+This page provides guidance exclusively for systems upgrading CSM v1.4.X to CSM v1.4.4.
+
+The [v1.4.4](../1.4.4/README.md) upgrade page will refer to this page
+during [Update NCN images](../1.4.4/README.md#update-ncn-images).
+
+**If other products are installed on the system, return to [Update NCN images](../1.4.4/README.md#update-ncn-images) and
+choose option 2.**
+
+## Requirements
+
+* `CSM_RELEASE` is set in the shell environment on `ncn-m001`.
+
+## Steps
+
+1. (`ncn-m001#`) Generate a new CFS configuration for the management nodes.
+
+   This script creates a new CFS configuration that includes the CSM version in its name and applies it to the
+   management nodes. This leaves the management node components in CFS disabled. They will be automatically enabled when
+   they are rebooted at a later stage in the upgrade.
+
+   ```bash
+   /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
+       --no-enable --config-name "management-${CSM_RELEASE}"
+   ```
+
+   Successful output should end with the following line:
+
+   ```text
+   All components updated successfully.
+   ```
+
+1. Set the IMS IDs for each Management node sub-role.
+
+    * Kubernetes ID
+
+      ```bash
+      KUBERNETES_IMAGE_ID="$(kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' \
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-kubernetes")) | $o[.].id')"
+      export KUBERNETES_IMAGE_ID
+      echo "KUBERNETES_IMAGE_ID=$KUBERNETES_IMAGE_ID"
+      ```
+
+    * Storage-CEPH ID
+
+      ```bash
+      STORAGE_IMAGE_ID="$(kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' \
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-storage")) | $o[.].id')"
+      export STORAGE_IMAGE_ID
+      echo "STORAGE_IMAGE_ID=$STORAGE_IMAGE_ID"
+      ```
+
+1. Create new CFS sessions for both Kubernetes and Storage nodes.
```

Comment:
You may want to have the session names include a timestamp. Or else include a note that these commands will fail if being run for a second time, because the CFS sessions already exist (in which case they would first need to be deleted)

Reply:
I added some additional comments with suggested changes to address this